docs/advanced-reading/fetchers.md

```diff
@@ -14,6 +14,42 @@ Fetchers are the implementation of the [search using online services](https://do
 
 On Windows, you have to log-off and log-on to let IntelliJ know about the environment variable change. Execute the gradle task "processResources" in the group "others" within IntelliJ to ensure the values have been correctly written. Now, the fetcher tests should run without issues.
 
+## Fetcher prioritization
+
+- all fetchers run in parallel
+- the result with the highest priority wins
+- `InterruptedException` | `ExecutionException` | `CancellationException` are ignored
+
+### Trust Levels
+
+- SOURCE (highest)
+- PUBLISHER
+- PREPRINT
+- META_SEARCH
+- UNKNOWN (lowest)
+
+### Current trust levels
+
+All fetchers are contained in the package `org.jabref.logic.importer.fetcher`.
+Here we list the trust levels of some of them:
+
+- DOI: SOURCE, because it is the ultimate source
+- ScienceDirect: Publisher
+- Springer: Publisher
```

Comment:
Might be wrong publisher or might search based on string similarity not always on DOI

Reply:
Also often requires an API_KEY and counts towards the api key usage limit